src/nasas.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>NASA-Homework</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="nasa.css">
+  <link href="https://fonts.googleapis.com/css?family=Bowlby+One+SC&display=swap" rel="stylesheet"> 
+</head>
+<body>
+  <main>
+    <h1>NASA</h1>
+  </main>
+  <section>
+    <article>Social Media</article>
+    <p class="sub1">Twitter - @nasa</p>
+    <p class="sub1">Facebook - fb.com/nasa</p>
+    <p class="sub1">Instagram - @nasa</p>
+    <p class="sub1">Youtube - youtube.com/nasa</p>
```

Comment:
we JUST talked about this! yes. wrap these in their own divs that way section flexbox will just affect three divs: social-media, office info, contact us.
`<div class="social">`
    `  <article>Social Media</article>
     ` <p class="sub1">Twitter - @nasa</p>`
     ` <p class="sub1">Facebook - fb.com/nasa</p>`
      `<p class="sub1">Instagram - @nasa</p>`
      `<p class="sub1">Youtube - youtube.com/nasa</p>`
    `</div>`

Reply:
Done